src/components/NewMessageModal.vue

```diff
@@ -392,7 +392,9 @@ export default {
 
 				if (!data.sendAt || data.sendAt < Math.floor((now + UNDO_DELAY) / 1000)) {
 					// Awaiting here would keep the modal open for a long time and thus block the user
-					this.$store.dispatch('outbox/sendMessageWithUndo', { id: dataForServer.id })
+					this.$store.dispatch('outbox/sendMessageWithUndo', { id: dataForServer.id }).catch((error) => {
+						logger.debug('Could not send message', { error })
+					})
```

Comment:
Because there is nothing to do about the error - sendMessageWithUndo handles the rejection of the promise if it fails. The error handling is to avoid the console errors for an uncaught in Promise exception from the rejection.

Reply:
Ah, and we can't try- catch because we can't await the store.dispatch. looked it through with richard and if we did, we would block the user from interactions until the 30 seconds of sending are done. So we have to do it the "ugly" way with a .catch on the promise